CallsheetLocalizations/uk.xcloc/Localized Contents/uk.xliff

```diff
@@ -37,1064 +37,1065 @@
+        <source>Callsheet was built in Virginia by **[Casey Liss](https://www.caseyliss.com/)**.</source><target>Callsheet було створено у Вірджинії **[Кейсі Ліссом](https://www.caseyliss.com/)**.</target>
         <note/>
       </trans-unit>
       <trans-unit id="Cancel" xml:space="preserve">
-        <source>Cancel</source>
+        <source>Cancel</source><target>Скасувати</target>
         <note/>
       </trans-unit>
       <trans-unit id="Cancel Subscription" xml:space="preserve">
-        <source>Cancel Subscription</source>
+        <source>Cancel Subscription</source><target>Скасувати передплату</target>
         <note/>
       </trans-unit>
       <trans-unit id="Cast" xml:space="preserve">
-        <source>Cast</source>
+        <source>Cast</source><target>Акторський склад</target>
         <note/>
       </trans-unit>
       <trans-unit id="Cast or Crew" xml:space="preserve">
-        <source>Cast or Crew</source>
+        <source>Cast or Crew</source><target>Акторський склад та знімальна група</target>
         <note/>
       </trans-unit>
       <trans-unit id="Choose New Subscription" xml:space="preserve">
-        <source>Choose New Subscription</source>
+        <source>Choose New Subscription</source><target>Оберіть нову підписку</target>
         <note/>
       </trans-unit>
       <trans-unit id="Choose a link:" xml:space="preserve">
-        <source>Choose a link:</source>
+        <source>Choose a link:</source><target>Оберіть ярлик:</target>
         <note>Shown in settings for the quick access link</note>
       </trans-unit>
       <trans-unit id="Choose a quick link" xml:space="preserve">
-        <source>Choose a quick link</source>
+        <source>Choose a quick link</source><target>Оберіть ярлик для швидкого доступу</target>
         <note>Accessibility label for the quick access picker in Persnickety Preferences</note>
       </trans-unit>
       <trans-unit id="Choose spoiler-risky things to hide by default." xml:space="preserve">
-        <source>Choose spoiler-risky things to hide by default.</source>
+        <source>Choose spoiler-risky things to hide by default.</source><target>Оберіть що приховувати за замовчуванням, щоб уникнути спойлерів</target>
         <note>Subtitle for a row in Settings</note>
       </trans-unit>
       <trans-unit id="Clear" xml:space="preserve">
-        <source>Clear</source>
+        <source>Clear</source><target>Очистити</target>
         <note/>
       </trans-unit>
       <trans-unit id="Clear all" xml:space="preserve">
-        <source>Clear all</source>
+        <source>Clear all</source><target>Очистити усе</target>
         <note>Shown when clearing recent searches or search history</note>
       </trans-unit>
       <trans-unit id="Click Me" xml:space="preserve">
-        <source>Click Me</source>
+        <source>Click Me</source><target>Натисни мене</target>
         <note>Do not translate</note>
```

Comment:
are you sure you need to translated this?

Reply:
https://github.com/cliss/callsheet-localizations/pull/3/commits/92d8298c05e7a90ebba434bb90661a256defdab3